plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -630,7 +639,34 @@ public void replaceTable(String databaseName, String tableName, Table newTable,
     @Override
     public void renameTable(String databaseName, String tableName, String newDatabaseName, String newTableName)
     {
-        throw new TrinoException(NOT_SUPPORTED, "Table rename is not yet supported by Glue service");
+        boolean newTableCreated = false;
+        try {
+            Optional<Table> table = getTable(databaseName, tableName);
+            if (table.isEmpty()) {
+                throw new TableNotFoundException(schemaTableName(databaseName, tableName));
+            }
+            TableInput tableInput = GlueInputConverter.convertTable(table.get())
+                    .withName(newTableName);
+            CreateTableRequest createTableRequest = new CreateTableRequest()
+                    .withDatabaseName(newDatabaseName)
+                    .withTableInput(tableInput);
+            stats.getCreateTable().call(() -> glueClient.createTable(createTableRequest));
+            newTableCreated = true;
+            deleteTable(databaseName, tableName);
```

Comment:
Why not `dropTable(databaseName, tableName, false);`?

Reply:
Because I like to write code so I write as much new methods as I can ;) 
Jokes aside  I didn't notice dropTable, thanks for pointing it out.